doc/src/manual/environment-variables.md

```diff
@@ -406,6 +406,14 @@ should have at the terminal.
 The formatting `Base.answer_color()` (default: normal, `"\033[0m"`) that output
 should have at the terminal.
 
+### `NO_COLOR`
+
+If set to anything besides `""`, then colored text will be disabled on the REPL. Can be overridden with the `--color=yes` flag. This environmental variable is [commonly recognized by command-line applications](https://no-color.org/).
```

Comment:
please wrap all lines, somewhere below 100 chars (or whatever the surrounding text uses)